osCam/templates/core/home.html

```diff
@@ -5,9 +5,8 @@
         <link rel="stylesheet" href="{% static 'core/dash.css' %}">
     </head>
 
-    {% include "core/navigation.html" %}
-
     <body>
+        {% include "core/navigation.html" %}
         <div class="text-center">
```

Comment:
Should core/navigation not be outside the body?

Reply:
wasn't sure on that one. shows up either way. idk best practice